metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -641,42 +572,23 @@ def parse_view_lineage(
         return list(parsed_tables)
 
     def _compute_bigquery_lineage(self, project_id: str) -> Dict[str, Set[LineageEdge]]:
-        lineage_extractor: BigqueryLineageExtractor = BigqueryLineageExtractor(
-            config=self.config, report=self.report
-        )
         lineage_metadata: Dict[str, Set[LineageEdge]]
         try:
             if self.config.extract_lineage_from_catalog and self.config.include_tables:
-                lineage_metadata = (
-                    lineage_extractor.compute_bigquery_lineage_via_catalog_lineage_api(
-                        project_id
-                    )
-                )
+                lineage_metadata = self.lineage_via_catalog_lineage_api(project_id)
             else:
-                if self.config.use_exported_bigquery_audit_metadata:
-                    # Exported bigquery_audit_metadata should contain every projects' audit metada
-                    if self.loaded_project_ids:
-                        return {}
-                    lineage_metadata = (
-                        lineage_extractor.compute_bigquery_lineage_via_exported_bigquery_audit_metadata()
-                    )
-                else:
-                    lineage_metadata = (
-                        lineage_extractor.compute_bigquery_lineage_via_gcp_logging(
-                            project_id
-                        )
-                    )
+                events = self._get_parsed_audit_log_events(project_id)
+                lineage_metadata = self._create_lineage_map(events)
         except Exception as e:
             if project_id:
                 self.report.lineage_failed_extraction.append(project_id)
-            logger.error(
-                f"Unable to extract lineage for project {project_id} due to error {e}"
+            self.error(
+                logger,
+                "lineage",
+                f"{project_id}: {e}",
```

Comment:
This is to replace the `self.error` calls in the deleted `compute_bigquery_lineage_via_gcp_logging` and `compute_bigquery_lineage_via_audit_metadata`. Since this is catching unexpected errors and has a relatively clean message, I think this is valid to put in the report.